unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneResourcesLoadTracker/Tests/SceneResourcesLoadTrackerShould.cs

```diff
@@ -1,100 +1,96 @@
-using DCL;
+using System.Collections;
+using System.Collections.Generic;
 using DCL.Controllers;
+using DCL.ECSComponents;
+using DCL.Models;
 using DCL.WorldRuntime;
 using NSubstitute;
+using NSubstitute.Extensions;
 using NUnit.Framework;
+using UnityEngine;
 
-namespace Tests
+namespace DCL.Test
```

Comment:
We already have those tests in 3 separated files

`SceneResourcesLoadTrackerShould ` is completely agnostic to the system, it doesn't use one or the other
`SceneResourcesLoadTrackerForECS7 ` that is the scene resource loader for the new ECS. It check the system with the new system
`ResourcesLoadTrackerLegacyECSShould` Check the resource loader system with the old system.

This way we can then delete the Legacy system without the need to touch anything.

Why we would only want to have the two last cases in one file?

Reply:
`ResourcesLoadTrackerLegacyECSShould` should test that ecs tracker
`SceneResourcesLoadTrackerForECS7` should test that ecs tracker
`SceneResourcesLoadTrackerShould` should test the tracker integration with `SceneResourcesLoadTracker` so we need to test that integration with both since both exists